src/borg/archiver.py

```diff
@@ -1121,11 +1121,22 @@ def item_to_tarinfo(item, original_path):
     def do_diff(self, args, repository, manifest, key, archive):
         """Diff contents of two archives"""
 
+        def actual_change(j):
+            if j["type"] != "modified":
+                return True
```

Comment:
Maybe add a comment here, guess without one this is not clear enough and may rather seem non-sensical:

"not modified" -> "yes, actual change!"

Reply:
I can reverse the `if`-`else` branches as well to make things more obvious, which in retrospect probably makes more sense.